cherab/openadas/repository/atomic.py

```diff
@@ -108,6 +108,47 @@ def update_recombination_rates(rates, repository_path=None):
         _update_and_write_adf11(species, rate_data, path)
 
 
+def add_thermalchargeexchange_rate(species, charge, rate, repository_path=None):
+    """
+    Adds a single thermal charge exchange rate to the repository.
+
+    If adding multiple rates, consider using the update_recombination_rates()
+    function instead. The update function avoids repeatedly opening and closing
+    the rate files.
+
+    :param repository_path:
+    :return:
+    """
+
+    update_thermalchargeexchange_rates({
+        species: {
+            charge: rate
+        }
+    }, repository_path)
+
```

Comment:
This nested dictionary should be more consistent with the other CX example, see this line
https://github.com/cherab/openadas/blob/9379b0651e8e69a3535137bfd959d51cbb44f232/cherab/openadas/repository/beam/cx.py#L47